src/core/unit.c

```diff
@@ -3759,6 +3758,13 @@ int unit_coldplug(Unit *u) {
         return r;
 }
 
+void unit_catchup(Unit *u) {
+        assert(u);
```

Comment:
Is it worth it to define this a separate function when there's just on user?

Reply:
i mostly capt it so that it matches nicely with unit_coldplug() which kinda does the same stuff...